.appveyor.yml

```diff
@@ -1,215 +1,143 @@
 image: Visual Studio 2015
 version: '{branch}.{build}'
 
-clone_folder: c:\projects\xdebug
+clone_folder: C:\projects\xdebug
 
 install:
-    ps: |
-        if (-not (Test-Path c:\build-cache)) {
-            mkdir c:\build-cache
-        }
-        $bname = 'php-sdk-' + $env:BIN_SDK_VER + '.zip'
-        if (-not (Test-Path c:\build-cache\$bname)) {
-            Invoke-WebRequest "https://github.com/OSTC/php-sdk-binary-tools/archive/$bname" -OutFile "c:\build-cache\$bname"
-        }
-        $dname0 = 'php-sdk-binary-tools-php-sdk-' + $env:BIN_SDK_VER
-        $dname1 = 'php-sdk-' + $env:BIN_SDK_VER
-        if (-not (Test-Path c:\build-cache\$dname1)) {
-            7z x c:\build-cache\$bname -oc:\build-cache
-            move c:\build-cache\$dname0 c:\build-cache\$dname1
-        }
+    ps: .appveyor\install.ps1
```

Comment:
I really prefer it all in one file, mostly so that I can see what you have actually changed. Especially the file name that AppVeyor generators is important to stay stable as I have other tooling around that. If it is necessary to have it in files, then in the first commit, move things to files; and in a second commit make the changes. (That also means that I'd like the current commits squashed).

Reply:
It is, of course, not necessary to factor the Powershell code to separate files.